### PR TITLE
feat: created typography component

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
- created typography component
- Added different text-sizes, text-alignment, text-boldness
- added code snippet using highlight js
Typography-preview1
Typography-preview2

### CODE REVIEW

File: components/typography.css [hunk 1]
```diff
@@ -1 +1,85 @@
-@import url(/stylesheets/main.css);;
\ No newline at end of file
+@import url(/stylesheets/main.css);
+@import url("https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/an-old-hope.min.css");
+
+/* font-sizes */
+.font-xxl {
+  font-size: 48px;
```
Comment (coldpigli) [inline components/typography.css:6]: why not rems?

File: stylesheets/general.css [hunk 2]
```diff
@@ -62,6 +62,6 @@ body{
 .parent-container{
     margin-top: 6rem;
     padding-left: 17rem;
-    width: 50rem;
+    /* width: 50rem; */
```
Comment (Sameony) [inline stylesheets/general.css]: remember to remove unused blocks of codes before merging

**Review by coldpigli — COMMENTED** [review]
Just a doubt. Everything else looks fine,

**Review by Sameony — APPROVED** [review]
very nice and clean code.

File: components/typography.css (hunk 1 above)
Comment (nakulsharma15) [inline components/typography.css:6]: You can use relative units(like rems) instead of px

File: components/typography.css [hunk 3]
```diff
@@ -1 +1,85 @@
...
+}
+
+.font-xl {
```
Comment (nakulsharma15) [inline components/typography.css:9]: Same as above

File: stylesheets/general.css (hunk 2 above)
Comment (vaibhav-mahalle) [reply stylesheets/general.css]: will keep that in mind

File: components/typography.css [hunk 4]
```diff
@@ -1 +1,85 @@
...
+.font-md {
+  font-size: 24px;
+}
+
+.font-sm {
+  font-size: 16px;
+}
+
+.font-xs {
+  font-size: 8px;
```
Comment (nakulsharma15) [inline components/typography.css:26]: Same as above

**Review by nakulsharma15 — COMMENTED** [review]
Suggested a minor change

File: components/typography.css (hunk 1 above)
Comment (Sameony) [reply components/typography.css:6]: https://stackoverflow.com/questions/11799236/should-i-use-px-or-rem-value-units-in-my-css
there's a war going on but each has its own cons and pros

Comment (vaibhav-mahalle) [reply components/typography.css:6]: no particular reason you want rems to make text responsive and but as of now I am not optimizing for that.

Comment (vaibhav-mahalle) [reply components/typography.css:6]: will make those changes while making library responsive

**Pull request merged**